android/src/org/ppsspp/ppsspp/NativeActivity.java

```diff
@@ -568,7 +573,6 @@ public void surfaceCreated(SurfaceHolder holder) {
 		}
 
 		Log.d(TAG, "Surface created. pixelWidth=" + pixelWidth + ", pixelHeight=" + pixelHeight + " holder: " + holder.toString() + " or: " + requestedOr);
-		NativeApp.setDisplayParameters(pixelWidth, pixelHeight, (int) densityDpi, refreshRate);
```

Comment:
there is a issue in latest version, touch zone doesn't match it's display position.
add back this line will fix it.
test on Mi9, Android Q.

Reply:
Thanks! I'll give that a shot, but so far every change we've done in this area has both fixed something and broken something else..